src/converters.ts

```diff
@@ -24,27 +26,50 @@ export class StringConverter<V> extends Converter<string, V> {
   defaultControlled = controlled.value;
 }
 
-const string = new StringConverter<string>({
-  emptyRaw: "",
-  emptyValue: "",
-  convert(raw) {
-    return raw;
-  },
-  render(value) {
-    return value;
-  },
-  preprocessRaw(raw: string): string {
-    return raw.trim();
-  }
-});
+type StringOptions = {};
 
-const number = new StringConverter<number>({
-  emptyRaw: "",
-  emptyImpossible: true,
-  convert(raw, converterOptions) {
-    checkConverterOptions(converterOptions);
-    try {
-      return +parseDecimal(raw, {
+function stringWithOptions(options?: StringOptions) {
+  return new StringConverter<string>({
+    emptyRaw: "",
+    emptyValue: "",
+    convert(raw) {
+      return raw;
+    },
+    render(value) {
+      return value;
+    },
+    preprocessRaw(raw: string): string {
+      return raw.trim();
+    }
+  });
+}
+
+const string = stringWithOptions();
```

Comment:
I like this idea of defining regular converters as converters with options without options. Clever.